code/controllers/subsystem/tremors.dm

```diff
@@ -0,0 +1,24 @@
+SUBSYSTEM_DEF(tremors)
```

Comment:
this has gotta use the weather subsystem.

Reply:
it would compete with ash storms, which i don't want because 
A: they are much less severe, lasting only ten seconds with no real damage
B: they'd nerf ash storms by making them spawn less often
C: to avoid B tremors would have to be rare, but it doesn't make sense due to A

Reply:
then make it possible to have two weather effects at once.

Reply:
guess i'll do that then

Reply:
as a side thought, you could always make ash storms occur directly *after* tremors; the seismic activity would stir the ash into the air, and the wind would pick it up and thus cause an ash storm. just my thoughts

Reply:
You could make it weighted as well, like we do for events